Find a functionally equivalent code:android.os.Bundle.getByte:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the value associated with the given key, or (byte) 0 if
 * no mapping of the desired type exists for the given key.
 *
 * @param key a String
 * @return a byte value
 */

@@ -1,4 +1,3 @@
 {
-    unparcel();
-    return getByte(key, (byte) 0);
+    return super.getByte(key);
 }

